committees/tests/test_committee_detail_view.py

```diff
@@ -132,15 +147,112 @@ def test_committeemeeting_by_tag(self):
             tag=self.tag_1,
             content_type=ContentType.objects.get_for_model(CommitteeMeeting),
             object_id=self.meeting_1.id)
-        res = self.client.get(reverse('committeemeeting-tag', args=[self.tag_1.name]))
+        res = self.client.get(
+            reverse('committeemeeting-tag', args=[self.tag_1.name]))
         self.assertEqual(res.status_code, 200)
-        self.assertTemplateUsed(res, 'committees/committeemeeting_list_by_tag.html')
+        self.assertTemplateUsed(res,
+                                'committees/committeemeeting_list_by_tag.html')
         tag = res.context['tag']
         self.assertEqual(tag, self.tag_1)
         self.assertQuerysetEqual(res.context['object_list'],
                                  ['<CommitteeMeeting: c1 - django>'])
-        res = self.client.get('%s?tagged=false' % reverse('committee-all-meetings'))
+        res = self.client.get(
+            '%s?tagged=false' % reverse('committee-all-meetings'))
         self.assertQuerysetEqual(res.context['object_list'],
                                  ['<CommitteeMeeting: c1 - python>'])
         # cleanup
         self.ti.delete()
+
+    def test_committee_all_members_return_all_types_of_related_members(self):
+        committee_3 = Committee.objects.create(name='c3')
+        chairperson_mk = Member.objects.create(name='Chief')
+        member_mk = Member.objects.create(name='member')
+        replacement_mk = Member.objects.create(name='replacement')
+        committee_3.members.add(member_mk)
+        committee_3.chairpersons.add(chairperson_mk)
```

Comment:
Please use setup methods "given" to encapsulate setup as above, also "verify" to encapsulate asserts.. 
More DRY but more important: makes the tests much more readable and less technical